__tests__/app.test.js

```diff
@@ -30,31 +30,69 @@ describe("Get 200: /api/categories", () => {
 });
 
 describe("404: response with a message not found url", () => {
-  test("404: response with a message not found for a typo", () => {
+  test("404: response with a message 'not found' for a typo", () => {
```

Comment:
I'd suggest changing this description to swap out 'typo' with something more precise: non-existent endpoint, or something like that ... let's not assume the mind of our user